<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectPropCal_Dialog</class>
 <widget class="QDialog" name="SectPropCal_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>354</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>354</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculation of section properties</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 9pt &quot;Segoe UI&quot;;
}

QDialog
{
	background-color: rgb(43, 43, 43);
}
QCheckBox
{
	color: rgb(255, 255, 255);
}
QCheckBox:disabled
{
	color: rgb(80, 80, 80);
}
QCheckBox::indicator:disabled
{
	background-color: rgb(160, 160, 160);
	border-radius: 3px;
}        
QComboBox
{
	border-radius: 3px;
}
QComboBox:hover
{
	background-color: rgb(244, 244, 244);
}
QComboBox::drop-down
{
	width: 20px;
	border-left: 1px solid rgb(192, 192, 192);
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}
QComboBox::down-arrow
{
	image:url(ui/ico/DownArrow.png);
	width: 10px;
}
QComboBox:disabled
{
	background-color: rgb(160, 160, 160);
}
QComboBox::drop-down:disabled
{
	border-left: 1px solid rgb(128, 128, 128);
}
QGroupBox
{
	background-color: rgb(128, 128, 128);
	color: rgb(255, 255, 255);
}
QLabel
{
	color: rgb(255, 255, 255);
}
QLabel:disabled
{
	color: rgb(80, 80, 80);
}
QLineEdit
{
	background-color: rgb(255, 255, 255);
	border-radius: 3px;
}
QLineEdit:hover
{
	background-color: rgb(244, 244, 244);
}
QLineEdit:disabled
{
	background-color: rgb(160, 160, 160);
	color: rgb(80, 80, 80);
}
QPushButton
{
	background-color: rgb(255, 255, 255);
	border-radius: 3px;
}
QPushButton:hover
{
	background-color: rgb(144, 200, 246);
}
QPushButton:pressed
{
    padding-left: 3px;
    padding-top: 3px;
}
QRadioButton
{
	color: rgb(255, 255, 255);
}
QRadioButton:disabled
{
	color: rgb(80, 80, 80);
}
QRadioButton::indicator:disabled
{
	height: 13px;
	width: 13px;
	background-color: rgb(160, 160, 160);
	border-radius: 6px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,3,3,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Analysis Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="CM_radioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Coordinate Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="FE_radioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Finite-Element Method</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Mat_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Equivalent Section Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="3" column="2">
       <widget class="QLineEdit" name="EquivPR_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>0.3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="EquivE_lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>205000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="Refmat_comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="UseDefValues_radioButton">
        <property name="text">
         <string>Use Defined Values:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>E (Equivalent Young's Modulus):</string>
        </property>
        <property name="indent">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Î¼ (Equivalent Poisson 's Ratio):</string>
        </property>
        <property name="indent">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>fy (Equivalent Design Strength):</string>
        </property>
        <property name="indent">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="UseRefMat_radioButton">
        <property name="text">
         <string>Use Reference Material ID:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="EquivDesiStrs_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>355</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Mesh_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Mesh Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="2" column="0">
       <widget class="QRadioButton" name="MeshSize_radioButton">
        <property name="minimumSize">
         <size>
          <width>208</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Mesh Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="UseExistingMesh_radioButton">
        <property name="text">
         <string>Use the Existing Mesh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="AutoMesh_radioButton">
        <property name="text">
         <string>Auto Mesh </string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ESMS_groupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Elastic Section Modulus Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="StrainatMaxiStress_radioButton">
        <property name="text">
         <string>Strain at the onset of Maximum Stress </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="StrainatValue_radioButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Strain at the Value of</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="StrainValue_lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>0.001</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>30</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Run_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cancel_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
